near-abi-client/tests/mod.rs

```diff
@@ -0,0 +1,42 @@
+use near_abi_client::Generator;
+use quote::quote;
+use std::fs;
+use tempdir::TempDir;
+
+#[test]
+fn test_generate_abi() -> anyhow::Result<()> {
+    let tmp_dir = TempDir::new("adder-generated-code")?;
+    let tmp_dir_path = tmp_dir.into_path();
+    Generator::new(tmp_dir_path.clone())
+        .file("tests/adder.json")
+        .generate()?;
+
+    let generated_code = fs::read_to_string(tmp_dir_path.join("adder.rs"))?;
+    let expected = quote! {
+        pub type Pair = Vec<i64>;
+        pub struct ExtAbi {
+            pub contract: workspaces::Contract,
+        }
+        impl ExtAbi {
```

Comment:
Also these tests are broken because of the new name
```suggestion
        pub struct AbiClient {
            pub contract: workspaces::Contract,
        }
        impl AbiClient {
```